build.gradle

```diff
@@ -656,27 +707,37 @@ if (jdkVersion == '8') {
         exclude group: 'org.pentaho' // missing dependency
         exclude group: 'org.slf4j', module: 'slf4j-log4j12'
       }
+
+      compileOnly("org.apache.orc:orc-core::nohive") {
```

Comment:
This kind of dependency w/ excludes is repeated. I think, it would be cleaner to add a new configuration named something like `orc-core-nohive` so so and add that configuration as a dependency instead of the repeated lines. WDYT?